### ISSUE TITLE
Analogue Pocket not working

### ISSUE THREAD

**Issue by Staacks** [Staacks/gbinterceptor#2]
This is a low priority item as the Analogue Pocket offers a dock with HDMI, so the GB Interceptor is not necessarily too important to Pocket users. Nevertheless, this should be fixable:
The Analogue Pocket shows a slightly different access pattern for the stack pointer during interrupt calls, which prevents the Interceptor's interrupt detection from triggering.

**Comment by xenyou** [comment]
That is the good news for me!!
I want to play pocket as handheld while recording the video.

**Comment by placidcasual98** [comment]
Yeah it's great that the dock has an output but what if you wanted to hold it while streaming? It's kind of hard to hold the dock. Don't want to use a different Bluetooth pad
